cronos-play/getting-started_unreal/quick-start/cronos-configuration.md

```diff
@@ -0,0 +1,14 @@
+# Cronos Configuration
+
+After creating **BP\_DefiWalletCore**, we should configure the Network Configuration, so that when we spawn the actor, we could get the same network settings.
+
+* Double click **BP\_DefiWalletCore** in **Content Browser**, and open the Blueprint Editor
+* In **Cronos Play Unreal** section of **Details** tab, input the RPC and Chain ID settings, for example like below:
+  * `My Cronos Rpc`: [https://cronos-testnet-3.crypto.org:8545](https://cronos-testnet-3.crypto.org:8545)
```

Comment:
Let use this rpc [evm-dev.cronos.org](http://evm-dev.cronos.org/) instead of normal RPC,
this RPC is specific for gameFi usage, so we could track the volume.

Reply:
do we have explorer for [evm-dev.cronos.org](http://evm-dev.cronos.org/)? @lezzokafka

Reply:
[evm-dev.cronos.org](http://evm-dev.cronos.org/) is mainnet, while the demo is testnet. It could be mentioned in the docs.

Reply:
My bad. We have 2 RPC. We should use the testnet one
[Mainnet] [evm-dev.cronos.org]
[Testnet] [evm-dev-t3.cronos.org]

Reply:
Done